Content.Server/Projectiles/Components/HitscanComponent.cs

```diff
@@ -20,37 +21,43 @@ namespace Content.Server.Projectiles.Components
     public class HitscanComponent : Component
     {
         [Dependency] private readonly IGameTiming _gameTiming = default!;
+        [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
 
         public override string Name => "Hitscan";
-        public CollisionGroup CollisionMask => (CollisionGroup) _collisionMask;
+
+        private TimeSpan _startTime;
+        private TimeSpan _deathTime;
 
         [DataField("layers")] //todo  WithFormat.Flags<CollisionLayer>()
         private int _collisionMask = (int) CollisionGroup.Opaque;
-
-        public float Damage
-        {
-            get => _damage;
-            set => _damage = value;
-        }
         [DataField("damage")]
         private float _damage = 10f;
-        public DamageType DamageType => _damageType;
-        [DataField("damageType")]
-        private DamageType _damageType = DamageType.Heat;
-        public float MaxLength => 20.0f;
 
-        private TimeSpan _startTime;
-        private TimeSpan _deathTime;
+        [DataField("damageType", required: true)]
+        private string _damageTypeID = default!;
+
+        private DamageTypePrototype _damageType => _prototypeManager.Index<DamageTypePrototype>(_damageTypeID);
 
-        public float ColorModifier { get; set; } = 1.0f;
-        [DataField("spriteName")]
-        private string _spriteName = "Objects/Weapons/Guns/Projectiles/laser.png";
         [DataField("muzzleFlash")]
         private string? _muzzleFlash;
         [DataField("impactFlash")]
         private string? _impactFlash;
         [DataField("soundHitWall")]
         private string _soundHitWall = "/Audio/Weapons/Guns/Hits/laser_sear_wall.ogg";
+        [DataField("spriteName")]
+        private string _spriteName = "Objects/Weapons/Guns/Projectiles/laser.png";
+
+
+        public DamageTypePrototype DamageType => _damageType;
+
+        public float MaxLength => 20.0f;
+        public CollisionGroup CollisionMask => (CollisionGroup) _collisionMask;
+        public float ColorModifier { get; set; } = 1.0f;
+        public float Damage
+        {
+            get => _damage;
+            set => _damage = value;
+        }
```

Comment:
The _damage field is redundant, you can make this an auto-property and apply the DataField attribute directly to it.